src/components/Order/Order.tsx

```diff
@@ -0,0 +1,167 @@
+import * as React from 'react';
+import styled from 'styled-components';
+import orderAction from '../../constants/orderAction';
+import orderOptions from '../../constants/orderOptions';
+import {OrderProps, OrderState} from './index';
+import OrderAction from './OrderAction/OrderAction';
```

Comment:
i don't see a big value for nesting these components in their own sub-directories? could you plz move them to a plain structure inside `Order` directory?

```
/Order
-- OrderAction
-- OrderButton
-- etc
```